storage/database/src/main/java/de/danoeh/antennapod/storage/database/DBUpgrader.java

```diff
@@ -349,6 +349,9 @@ static void upgrade(final SQLiteDatabase db, final int oldVersion, final int new
             db.execSQL("ALTER TABLE " + PodDBAdapter.TABLE_NAME_FEED_ITEMS
                     + " ADD COLUMN " + PodDBAdapter.KEY_PODCASTINDEX_TRANSCRIPT_TYPE + " TEXT");
         }
+        if (oldVersion < 3060000) {
```

Comment:
If i remember correctly, we need to store three states on the feed item itself.
0=DL Disabled
1=Follow global
2=DL Enabled

If we don't differentiate between 1 and 2, it will not be possible for single feeds to be "undecided". They would be either enabled, or disabled. Though i'll check this again

Reply:
Sorry, what I meant was: the 3 states are fine, but I don't think the database values of existing users need to be migrated. For most users they already are "1", so they would default to the global default. No need to manually migrate them to "2" (overwrite global with "on")

Reply:
Ah! Sure, that's possible.

My initial idea was that since we don't know the users intent, we just force enable them again after the migration, this way they will always be downloaded. We might download too much, but not too few.

Your suggestion works too, but since the new default is "off", we don't download anything, not even formerly enabled entries. However, if we migrate the "autodownload enabled" variable to the new one, that will work out to be exactly the same set of downloads than before. 

That sounds better to me than my implementation. What do you think?

Reply:
Yes, that's what I meant. Keep the behavior for existing users the same. I think we could even use the existing preference, no need to create a new one with the name "global"